src/client/src/term/disclaimerDialog.tsx

```diff
@@ -30,8 +30,17 @@ export const DisclaimerDialog = ({ markdownContent, onClose = () => {} }: Discla
   };
 
   return (
-    <Dialog open={open} onClose={handleClose}>
-      <Stack sx={{ width: 500, borderRadius: 1 }}>
+    <Dialog
+      open={open}
+      onClose={handleClose}
+      maxWidth="md"
+      PaperProps={{
```

Comment:
REQ: `PaperProps` ist in der [online-doku als deprecated markiert](https://mui.com/material-ui/api/dialog/#dialog-prop-PaperProps). Use `slotProps.paper` instead. Braucht es das überhaupt, mit dem maxWidth property?